src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -1,28 +1,68 @@
 package frc.robot.subsystems;
 
 import com.revrobotics.CANSparkMax;
-import com.revrobotics.SparkMaxPIDController;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 import PIDControl.PIDControl;
-import edu.wpi.first.math.controller.PIDController;
+import PIDControl.PIDControl.Coefficient;
 import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj.DutyCycleEncoder;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class Arm extends SubsystemBase 
 {
-    private DigitalInput _limitSwitch;
+    public static Arm _instance;
+
+    public static Arm getInstance()
+    {
+        if(_instance == null) 
+        {
+            _instance = new Arm();
+        }
+
+        return _instance;
+    }
+
+    private DigitalInput     _limitSwitch;
     private DutyCycleEncoder _pitchEncoder;
-    private CANSparkMax _linearMotor;
-    private CANSparkMax _pitchMotor;
-    private PIDControl _shoulderPid;
-    private boolean _extensionPidActive;
-    private PIDControl _extensionPid;
-    
+    private CANSparkMax      _linearMotor;
+    private CANSparkMax      _pitchMotor;
+    private CANSparkMax      _followerPitchMotor;
+    private PIDControl       _shoulderPid;
+    private boolean          _extensionPidActive;
+    private PIDControl       _extensionPid;
+    private RelativeEncoder  _extensionEncoder;
 
```

Comment:
This is just a general thing that we've done in the past but with the pid's, we've written it as _shoulderPID

Reply:
Something like this isn’t crucial year-to-year. Let’s just make sure we keep it consistent for an individual robot